quantum/nodes/diagram-output/diagram-output.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+
+  function DiagramOutputNode(config) {
+    RED.nodes.createNode(this, config);
+    const node = this;
+
+    node.on('input', async (msg, send, done) => {
+      console.log(`diagram-output: ${msg}`);
+      let shellScript = snippets.PRINT_DIAGRAM;
+      await shell.execute(shellScript, (err, data) => {
+        if (err) node.error(err);
+        // print the diagram to debug console
+        node.warn(data);
+        console.log(data);
+      });
+      shell.stop();
+      // pass the quantum register config to the output
+      send(msg);
```

Comment:
```suggestion
      await shell.execute(shellScript, (err, data) => {
        if (err) {
          done(err);
        } else {
          msg.payload = data;
          send(msg);
          done();
        }
      });
```
Same thing as above with `console.log`. With `node.warn`, the diagram isn't a warning message so it's improper use of the function. Ideally what we should do is send it as a payload to the next node, then the user can choose what they want to do with the diagram (send it to a file, as input for another function, etc), rather than being limited to just the debug console.

We also don't want to run `shell.stop()` here, as doing so will break any nodes which come after it that use the Python shell. We can't know for sure when a users flow will definitely end, so using it like this is likely to cause problems.

Reply:
The diagram output node is still a WIP, I accidentally commited it in this PR so might just exclude this file in this PR

Reply:
> The diagram output node is still a WIP, I accidentally commited it in this PR so might just exclude this file in this PR

Ah I see, and yeah I think it would be ideal if you could remove it from this PR and place it in a separate one.